app/views/events/_past_events.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if @past_events.any? %>
+	<div class="row events-header">
+		Past Events
+	</div>
+
+	<div>
+		<% @past_events.each do |event| %>
+            <div class = "row event-row"> 
+           
+        
+			<p class= "event-creator">Created By : <%= event.user.name %> </p>
+            <p class= "event-description">Description : <%= event.description %></p>
+			<p class= "event-date">Date & Time :	<%= event.event_date %></p>
+	
+		
+		    
+            <% if logged_in? %>
+				<p class= "event-delete">  <button><%= link_to "delete", event, method: :delete, 

```

Comment:
only the owner of the post should see the delete button

Reply:
this is rectified